modules/system/rolls/channel-test.js

```diff
@@ -149,11 +171,26 @@ export default class ChannelTest extends TestWFRP {
     if (this.context.previousResult?.SL > 0)
       SL -= this.context.previousResult.SL
 
+
+    //@HOUSE
+    if(this.preData.unofficialGrimoire && (this.item.cn.SL + SL) > this.item.cn.value) {
+      game.wfrp4e.utility.logHomebrew("unofficialgrimoire-overchannelling");
+      this.result.overchannelling = this.item.cn.SL + SL - this.item.cn.value;
+    }
+    //@HOUSE
+  
+    if(SL > 0) {
+      this.result.SL = "+" + SL;
+    } else {
+      this.result.SL = SL.toString()
+    }
```

Comment:
This is outside the homebrew sections, is it a fix for something?

Reply:
yes, as some other aspects are modifying the SLs, but never update the value in the result object, and this leads to different values shown on card vs actually applied, i.e. this part: 

      if (Number(SL) == 0 && game.settings.get("wfrp4e", "extendedTests"))
          SL = 1;